target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,14 @@ engenius,ens620ext)
 	ucidef_set_led_netdev "lan1" "LAN1" "green:lan1" "eth0"
 	ucidef_set_led_netdev "lan2" "LAN2" "green:lan2" "eth1"
 	;;
+mikrotik,ldf-5-ac)
+	ucidef_set_led_wlan "wlan5g" "WLAN" "green:wlan5g" "phy0tpt"
```

Comment:
Please use DT trigger instead.

Reply:
Tnx. Puzzled by your comment though. 'phy0tpt' is what is defined in the device tree for this LED. Other 01_leds files do the same. Please clarify what your request is.

Reply:
One does either define a trigger in DT or here. You did both. Just drop the wlan5g line here in 01_leds and keep the DT trigger.